### PR TITLE
[MAINTENANCE] Remove add_sorters methods

### PR DESCRIPTION
**Pull request by tyler-hoffman**
`DataAsset::add_sorters` is no longer used, as partitioning/sorting is now a responsibility of the batch definition.

- [ ] Description of PR changes above includes a link to [an existing GitHub issue](https://github.com/great-expectations/great_expectations/issues)
- [ ] PR title is prefixed with one of: [BUGFIX], [FEATURE], [DOCS], [MAINTENANCE], [CONTRIB]
- [ ] Code is linted - run `invoke lint` (uses `ruff format` + `ruff check`)
- [ ] Appropriate tests and docs have been updated

For more information about contributing, see [Contribute](https://docs.greatexpectations.io/docs/contributing/contributing_checklist).

After you submit your PR, keep the page open and **monitor the statuses of the various checks made by our continuous integration process at the bottom of the page. Please fix any issues that come up** and [reach out on Slack](https://greatexpectations.io/slack) if you need help. Thanks for contributing!

### CODE REVIEW

File: docs/docusaurus/versioned_docs/version-0.17/components/reference_notebooks/_fluent_datasources_assumed_code_for_guides.ipynb [hunk 1]
```diff
@@ -44,17 +44,17 @@
   },
   {
    "cell_type": "markdown",
-   "source": [
```
Comment (tyler-hoffman) [inline docs/docusaurus/versioned_docs/version-0.17/components/reference_notebooks/_fluent_datasources_assumed_code_for_guides.ipynb]: I need to revert the changes in this file

**Review by cdkini — APPROVED** [review]
